livedata/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -607,7 +608,7 @@ class ChatDomainImpl private constructor(
         // store the messages
         repos.messages.insertMessages(messages)
 
-        logger.logI("stored ${channelsResponse.size} channels, ${configs.size} configs, ${users.size} users and ${messages.size} messages")
+        logger.logI("storeStateForChannels stored ${channelsResponse.size} channels, ${configs.size} configs, ${users.size} users and ${messages.size} messages")
```

Comment:
Is this some left over code?

Reply:
It's the method name, indicating _what_ stored those channels. Think "the `storeStateForChannels` method stored..."